### ISSUE TITLE
Adding a template to refactored note [enhancement]

### ISSUE THREAD

**Issue by archelpeg** [lynchjames/note-refactor-obsidian#24]
Currently I write notes on a subtopic in my topic note and refactor them when I have enough for a subtopic note. Now I have to manually add a template and a link to the topic in the subtopic note.

It would be great if the refactoring could add a template automatically to the refactored note.
In the template I would like to include a placeholder. This placeholder should then be transformed to a link to the note from which I performed the refactoring action. (So more functionality than the standard Obsididan templates.)

Would this be possible?

**Comment by lynchjames** [comment]
So, the template for the content replacing the refactored note content in the original note could be something like this?

```
**Related Idea**
[[{{new_note_title}}]] #refactored
```

OR

```
#### See also

![[{{new_note_title}}]]
```

**Comment by archelpeg** [comment]
```
# {{title}} <!-- this functionality is already implemented with the "first line of selected text as heading -->


<here is where the refactored text goes in the new note>


&emsp;

---

**↑:**
[[<link to note from which I performed the refactor>]]
**vgl.:**

**Verweis:**

**Status:**
#
**Datum:**
{{date}}
**Links zu dieser Seite:**
```

This is my template for creating new notes . Under the arrow I always link to the note that is above it in the hierarchy, so in this case where I performed the note refactor. Currently I do this by hand.

In fact I don't want to add a template to the original note. 
The current behaviour of inserting a link to the refactored note is not my optimal workflow, I mostly move it to the end of my original note (that's how I link downwards in the hierarchy).
I'm not sure whether it would be a lot of work to make a toggle for it to either insert the link to where the refactoring happened or to the end of the original note.
@mrjackphil's plugin (https://github.com/mrjackphil/obsidian-crosslink-between-notes) has this behaviour when linking a note up in the hierarchy by pressing a hotkey, that is what I would like to happen when I refactor a note.

**Comment by lynchjames** [comment]
Got it, thanks for the additional details. I think I have all I need to implement.

**Comment by lynchjames** [comment]
Available in version 1.5.0.
